submissions/mustbefail/html-movie-seat-booking/style.css

```diff
@@ -0,0 +1,307 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v7.0.29,
+* Autoprefixer: v9.7.6
+* Browsers: last 4 version
+*/
+
+body {
+  background: #232433;
+}
+
+main {
+  font-family: Roboto, 'sans-serif';
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+main * {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.desctop {
+  display: none;
+}
+
+.booking {
+  color: #eee;
+  background: #232433;
+  padding: 0 10px;
+  position: relative;
+  text-align: center;
+  z-index: 1;
+}
+
+.booking:after {
+  content: '';
+  background-image: url(https://www.themoviedb.org/t/p/original/v3wGDRhnik4HSuaMzATkgaqDJLS.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  background-blend-mode: color;
+  opacity: 0.09;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+.booking h1 {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+.tagline {
+  margin-top: 0;
+  font-style: italic;
+  text-align: right;
+  color: rgb(182, 182, 182);
+}
+
+.info-block {
+  padding: 5px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  color: rgb(214, 214, 214);
+  line-height: 1.8;
+}
+
+.info-block span {
+  background: #eee;
+  color: #333;
+  padding: 2px 5px;
+  margin: 0 2px;
+  border-radius: 3px;
+  white-space: nowrap;
+  height: 25px;
+}
+
+.info-block .rate {
+  border: 1px solid #eee;
+  font-weight: bold;
+  text-shadow: 1px 1px rgba(219, 219, 219, 0.5);
+  background: rgb(156, 38, 38);
+  color: #eee;
+  padding: 1px 5px;
+}
+
+.info-block img {
+  width: 35%;
+  height: 35%;
+}
+
+.rating {
+  margin-bottom: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.rating-badge {
+  position: relative;
+  width: 45px;
+  height: 45px;
+  margin-right: 15px;
+  background: rgb(6, 192, 115);
+  line-height: 45px;
+  font-weight: bold;
+  border-radius: 50%;
+  overflow: hidden;
+}
+
+.rating span {
+  background: transparent;
+  color: #eee;
+}
+
+button {
+  background: transparent;
+  border: 1px solid #eee;
+  padding: 7px 17px;
+  color: #eee;
+  border-radius: 3px;
+  -webkit-transition: background 0.1s;
+  -o-transition: background 0.1s;
+  transition: background 0.1s;
```

Comment:
For better visual effect add easing function, not required but visual representing will be much better.